tests/test_subsample.py

```diff
@@ -70,17 +70,17 @@ def test_subsample(self):
                 self.assertEqual(
                     len(subsampled_values),
                     sample_size,
-                    f'Amount of values is incorrect. Expecting a sample size of {sample_size}',
+                    f"Amount of values is incorrect. Expecting a sample size of {sample_size}",
```

Comment:
Tangentially: "number of" is preferable to "amount of" here, because "values" is a count noun (how many) instead of a mass noun (how much).  More [here](http://crosstalk.cell.com/blog/the-difference-between-number-and-amount.).

```suggestion
                    f"Number of values is incorrect. Expecting a sample size of {sample_size}",
```